x-pack/qa/repository-old-versions/src/test/java/org/elasticsearch/oldrepos/DocValueOnlyFieldsIT.java

```diff
@@ -198,16 +198,5 @@ public void setupIndex() throws IOException {
         createRestoreRequest.addParameter("wait_for_completion", "true");
         createRestoreRequest.setJsonEntity("{\"indices\":\"" + indexName + "\"}");
         assertOK(client().performRequest(createRestoreRequest));
-
-        // add mappings (they will be auto-converted later)
-        Request putMappingsRequest = new Request("PUT", "/" + indexName + "/_mappings");
-        XContentBuilder mappingsBuilder = XContentFactory.jsonBuilder().startObject().startObject("properties");
-        for (String type : basicTypes) {
-            mappingsBuilder.startObject(type).field("type", type).endObject();
-        }
-        mappingsBuilder.startObject("date").field("type", "date").field("format", "yyyy/MM/dd").endObject();
-        mappingsBuilder.endObject().endObject();
-        putMappingsRequest.setJsonEntity(Strings.toString(mappingsBuilder));
-        assertOK(client().performRequest(putMappingsRequest));
```

Comment:
Can you clarify why this needs to be removed?

Reply:
This is now automated by the mapping conversion. Prior to this PR, you had to manually define the mapping after importing a legacy index in order to access the fields in that index (the original mapping would only be imported to _meta/legacy-mappings section).